internal/pkg/template/templates/workloads/services/static-site/cf.yml

```diff
@@ -283,6 +283,18 @@ Resources:
                     # Required otherwise ContentType won't be applied.
                     # See https://github.com/aws/aws-sdk-js/issues/1092 for more.
                     MetadataDirective: "REPLACE"
+          InvalidateCache:
+            Type: Task
+            End: true
+            Resource: arn:aws:states:::aws-sdk:cloudfront:createInvalidation
+            Parameters:
+              DistributionId: !Ref CloudFrontDistribution
+              InvalidationBatch:
+                CallerReference.$: States.UUID()
```

Comment:
Should the param be `CallerReference` instead of `CallerReference.$`? Also how does `States.UUID()` work?

Reply:
`States.UUID()` is an intrinsic function within the Amazon States Language: https://docs.aws.amazon.com/step-functions/latest/dg/amazon-states-language-intrinsic-functions.html#asl-intrsc-func-uuid-generate

"To use intrinsic functions you must specify .$ in the key value in your state machine definitions"